src/AutoPilotPlugins/PX4/CameraComponentSummary.qml

```diff
@@ -14,7 +14,7 @@ FactPanel {
     QGCPalette { id: qgcPal; colorGroupEnabled: enabled }
     FactPanelController { id: controller; factPanel: panel }
 
-    property Fact _camTriggerMode:  controller.getParameterFact(-1, "TRIG_MODE")
+    property Fact _camTriggerMode:  controller.getParameterFact(-1, "TRIG_MODE", false)
```

Comment:
Working on it. I'm going through a do-over on general UI problems. Mostly the very egregious one first, especially when it comes to the iPhone. I will add this in.